components/utils.py

```diff
@@ -810,6 +822,11 @@ def _get_upgrade_properties(self, service_name):
         with open(upgrade_props_file) as f:
             return json.load(f)
 
+    # XXX reconcile these two with the definitions above
+    def get_install_properties(self, service_name):
```

Comment:
Due to our time constraints, we can leave it as is and open a JIRA for it. Please change XXX to TODO